<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="6,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="QgraphicsView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,0">
          <item>
           <widget class="QPushButton" name="pushButton_PrevImage">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Previous Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_file_path">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>File Path:</string>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_imageNumber">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_NxtImage">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Next Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_Enhance">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Enhance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Binarized">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Binarized</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_DrawBoxes">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Draw Boxes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_OriginalImg">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Original</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,8,0,0,0,0">
        <item>
         <widget class="QLabel" name="label_file_name">
          <property name="text">
           <string>File Name</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>Region</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Label</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Database">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Use Temporary Databse? </string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_1">
          <item row="5" column="0">
           <widget class="QPushButton" name="pushButton_LabelPrespore">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Prespore (2)</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="pushButton_Label2cells">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>2 cells (4)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButton_LabelOther">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Other (0)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_StartLabeling">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Start Labeling (Space)</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QPushButton" name="pushButton_Labelmulticells">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>multiple cells (6)</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QPushButton" name="pushButton_PreviousROI">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Previous Region</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton_LabelCell">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Cell (1)</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="pushButton_LabelNotSure">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Not sure (9)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="pushButton_LabelSpore">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Spore (3)</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QPushButton" name="pushButton_NextROI">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Next Region</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="pushButton_Label3cells">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>3 cells (5)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox_AutoZoom">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Autozoom</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_CurrentLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Current Label:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
          <item row="5" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Spores</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QSpinBox" name="spinBox_numPrespores">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_current_cells">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QSpinBox" name="spinBox_numSpores">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Cells</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>ManualCount</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Prespores</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="spinBox_numCells">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Cell Counts</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_current_prespores">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_current_spores">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>FromLabels</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1195</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionOpen_Folder"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset>
     <normaloff>../Icons/home-icon.png</normaloff>../Icons/home-icon.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionLabel">
   <property name="icon">
    <iconset>
     <normaloff>../Icons/tag-icon.png</normaloff>../Icons/tag-icon.png</iconset>
   </property>
   <property name="text">
    <string>Label</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
